target.json

```diff
@@ -33,9 +72,6 @@
     "armv7-m",
     "arm",
     "gcc",
-    "lwip",
```

Comment:
You're right, it's absolutely an unrelated change and it will break things.  I'll add another revision to fix it.